src/Experiment/src/SignatureHelper.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Ecma335;
+
+namespace System.Reflection.Emit.Experimental
+{
+    //This is prototype code, to generate simple signatures.
+    //For more complex signatures, port System.Reflection.Emit's SignatureHelper.
+    internal static class SignatureHelper
+    {
+        internal static BlobBuilder MethodSignatureEnconder(ParameterInfo[]? parameters, ParameterInfo? returnType, bool isInstance)
```

Comment:
```suggestion
        internal static BlobBuilder MethodSignatureEncoder(ParameterInfo[]? parameters, ParameterInfo? returnType, bool isInstance)
```

Several places have the same typo.
```suggestion
        internal static BlobBuilder MethodSignatureEnconder(ParameterInfo[]? parameters, ParameterInfo? returnType, bool isInstance)
```